python/kvikio/pyproject.toml

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
+# See file LICENSE for terms.
+
+[build-system]
+build-backend = "scikit_build_core.build"
+requires = [
+    "cmake>=3.26.4",
+    "cython>=3.0.0",
+    "ninja",
+    "scikit-build-core[pyproject]>=0.7.0",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project]
+name = "kvikio-cu11"
+dynamic = ["version"]
+description = "KvikIO - GPUDirect Storage"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "Apache 2.0" }
+requires-python = ">=3.9"
+dependencies = [
+    "cmake>=3.26.4",
+    "cython>=3.0.0",
+    "ninja",
+    "scikit-build-core[pyproject]>=0.7.0",
```

Comment:
These are wrong, but they should get fixed once you apply my suggestions to dependencies.yaml.